src/libANGLE/Context.cpp

```diff
@@ -3568,6 +3568,11 @@ void Context::initCaps()
         mState.mCaps.maxSmoothLineWidth            = 1.0f;
     }
 
+    if (getClientType() == EGL_OPENGL_API || getClientVersion() >= Version(3, 0))
```

Comment:
In theory it could be supported in 2.0 too

Reply:
Probably yes. Will change.